### PR TITLE
Ft:delete-contact

### PR DESCRIPTION
**Pull request by sostene12**
#### What does this PR do?
-implement delete-contact
#### Description of Task to be completed
- get contact(sent contact from the user) then delete by id
-return a message as a response otherwise return an error message
#### How should this be manually tested?
-postman
#### What are the relevant pivotal tracker stories?
-delete-contact message

### CODE REVIEW

**Comment by wakaflorien** [comment]
> 

> Resolve the conflicts @sostene12 
### How should this manually be tested should be something like:
`clone the repo using git clone repo`
`change branch git checkout branch_name`
`make a POST/GET request on route_name`

File: controllers/contactController.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import Contact from "../models/contact";
+
+class ContactController{
+    static async createContact(req,res){
+        try {
+            const contact = new Contact(req.body);
+            await contact.save();
+            res.status(201).json(contact);
```
Comment (wakaflorien) [inline controllers/contactController.js:8]: use the same json form in responses. be consistent
{status, message, data}

**Review by wakaflorien — CHANGES_REQUESTED** [review]
correct those on your next commit.

File: controllers/contactController.js (hunk 1 above)
Comment (sostene12) [reply controllers/contactController.js:8]: let me change the form

**Pull request merged**